benchexec/util.py

```diff
@@ -123,6 +123,20 @@ def decode_to_string(toDecode):
         return toDecode
 
 
+def decode_path(path):
```

Comment:
I would prefer to keep `decode_path` in `container.py` (likely as `_decode_path`). It is good to keep utilities close to where they are used, if they are not globally useful. `util.py` is already quite some kitchensink of various stuff.